apps/omg_watcher/test/block_getter/core_test.exs

```diff
@@ -317,6 +317,7 @@ defmodule OMG.Watcher.BlockGetter.CoreTest do
     |> assert_check([1_000])
   end
 
+  @tag :capture_log
```

Comment:
is this a leftover?

Reply:
no, intended, one case issues a `warn` log

Reply:
ok